### DISCUSSION TITLE
Pre selecting data table rows and showing batch actions on load

### DISCUSSION THREAD

**Discussion by number-00** [carbon-design-system/carbon#13038] [Help]
How to pre select first two rows and show batch actions only when at least 2 rows are selected as soon as table loads?
Once the table loads, users can change the selection and select any other rows.
I have made a codesandbox with the user selection, but how to preselect the first two rows and show batch actions on load?

https://codesandbox.io/s/magical-farrell-uyc8ek?file=/index.js

**Comment by tay1orjones** [comment]
Rows can be preselected by adding `isSelected: true` to the row in your `rows` data.

https://github.com/carbon-design-system/carbon/blob/19fea2e26e26bf71064ae2d32f8691e14a2d1426/packages/react/src/components/DataTable/DataTable.js#L97-L110
